packages/smart-wallet/src/smartWallet.js

```diff
@@ -366,6 +428,87 @@ export const prepareSmartWallet = (baggage, shared) => {
             },
           });
         },
+
+        provideBrandToPurses() {
```

Comment:
the caller needn't be concerned with provisioning. shouldn't this just be `getBrandToPurses`?

I'm also wondering whether it's worth putting it on the facet vs just inlining. What else would use this?

I'm genuinely not sure. It is a "helper" method so maybe it doesn't matter much.

Reply:
I'm hoisting it to an ordinary function.

inlining obscured the code in `providePurseForKnownBrand`, but yeah, a facet method is overkill.